app/javascript/app/components/ndcs/lts-explore-table/lts-explore-table-selectors.js

```diff
@@ -167,3 +151,19 @@ export const getFilteredData = createSelector(
     });
   }
 );
+
+export const getFilteredDataBySearch = createSelector(
+  [getFilteredData, getQuery],
+  (data, query) => {
+    if (!data || isEmpty(data)) return null;
+    return data.filter(d => {
+      let match = false;
+      Object.keys(d).forEach(col => {
+        if (deburrUpper(d[col]).indexOf(query) > -1) {
+          match = true;
+        }
+      });
+      return match;
```

Comment:
We're repeating this in 3 places, do you think we can extract this bit and move it to the `utils` file as a filter-by-query function? Or entire selector if possible :)

Reply:
Yeah, we cant extract the selector but I will create an utils function

Reply:
Thanks :+1: